quarkchain/core.py

```diff
@@ -856,6 +858,7 @@ class RootBlockHeader(Serializable):
         ("extra_data", PrependedSizeBytesSerializer(2)),
         ("mixhash", hash256),
         ("signature", FixedSizeBytesSerializer(65)),
+        ("total_difficulty", biguint),
```

Comment:
I would suggest moving total_difficulty field just after difficulty field - in general, we want to leave signature to be the last field in the header.

Reply:
done